test/strategies/Jamfile.v2

```diff
@@ -38,6 +38,7 @@ test-suite boost-geometry-strategies
     [ run thomas.cpp                         : : : : strategies_thomas ]
     [ run transform_cs.cpp                   : : : : strategies_transform_cs ]
     [ run transformer.cpp                    : : : : strategies_transformer ]
+    [ run matrix_transformer.cpp             : : : : matrix_strategies_transformer ]
```

Comment:
`strategies_matrix_transformer`

FYI, the whole point of prefixing all of the tests with the same word is to allow our regression tool to filter tests and put them in separate sections in the regression matrix, see:
https://www.boost.org/development/tests/develop/developer/geometry.html

Reply:
Thanks for that pointer, I was unaware of that and I will fix it.